### DISCUSSION TITLE
Birds detected as people

### DISCUSSION THREAD

**Discussion by lethalMango** [blakeblackshear/frigate#2912] [Ask A Question]
I have a several cameras which are picking up birds as people.

The camera is an 8MP camera but covers a large FOV and requires the higher res for detection/usable recognition at longer distances.

I've seen the various advanced options in the reference config for specifying sizes. What's the best way to filter these out without affecting person detection?

![Magpie](https://user-images.githubusercontent.com/1101765/156944615-e9ab913d-daf7-4875-afa5-3e2fbf0f4c4c.png)
![Pigeon](https://user-images.githubusercontent.com/1101765/156944617-328f572a-05e7-4b2f-91cc-b45e16d1dc5d.png)
![Duck](https://user-images.githubusercontent.com/1101765/156944663-de7cad12-842c-4345-b297-c7e0a24e438c.png)

**Comment by NickM-27** [comment]
The best way would be to add a min_area filter for person to your config. It depends on the resolution of your camera, but I would first try seeing what the area of the bird is and then doubling it and setting that and see how it goes.

**Comment by Joshndroid** [comment]
I have the same problem and a similar resolution camera.. What area did you end up setting for min_area @lethalMango ?

**Reply by NickM-27** [reply]
It really depends on the resolution of your detect stream in frigate

**Reply by Joshndroid** [reply]
> It really depends on the resolution of your detect stream in frigate

Hey mate, I am using the default 1280x720 on a 4K camera... Its hilarious because every now and then I will get a hit for an off cut of a palm tree as a person at 70% haha.. i upped the threshold to 75 however some of the birds have been 76 and i dont want to go too high as sometimes even people are listed as 76%

**Reply by NickM-27** [reply]
If you have any real detections of people on that camera compare the snapshot with bounding boxes (that will also have the area) to the snapshot with bounding boxes (and area) of the bird and somewhere in between there is a good min_area
